kotlin-js/src/jsMain/kotlin/js/function/JsConstructorFunction.adapter.kt

```diff
@@ -0,0 +1,8 @@
+package js.function
+
+import js.array.*
+
+fun <C: JsClass<C>> JsConstructorFunction(
+    fn: C.() -> C,
```

Comment:
For start it can be unsafe.
Single parameter - `JsClass`

Reply:
```kt
fun <C: JsClass<C>> JsConstructorFunction(
    clazz: JsClass<C>,
): JsConstructorFunction<JsTuple, C> = clazz.unsafeCast<JsConstructorFunction<JsTuple, C>>()
```
like this?

Reply:
Type parameters should be the same like in `JsConstructorFunction` declaration.
Same order, same names.

Reply:
Like this?
```kt
fun <A : JsTuple, R : Any> JsConstructorFunction(
    clazz: JsClass<R>,
): JsConstructorFunction<A, R> = clazz.unsafeCast<JsConstructorFunction<A, R>>()
```

Reply:
Yes